### DISCUSSION TITLE
Understanding transport algorithms' source code

### DISCUSSION THREAD

**Discussion by jundonglin** [PMEAL/OpenPNM#2885] [Q&A]
I have some confusion when reading the source code of the Transport algorithm, can you recommend some relevant literature or examples for my reference

**Comment by ma-sadeghi** [comment]
It's basically conservation laws. If you could exactly point out which part you don't understand, we might be able to help.

**Reply by jgostick** [reply]
There is also the docs: https://openpnm.org/examples/reference/simulations/basic_transport.html
